projects/assets/src/translations/en/asm.ts

```diff
@@ -24,5 +24,10 @@ export const asm = {
     },
     csagentTokenExpired: 'Your customer support agent session is expired.',
     endSession: 'End Session',
+    sessionTimer: {
+      label: 'Session Timeout',
+      minutes: 'min',
```

Comment:
maybe min(s) because it can be 1 min or 10 mins

EDIT: no idea now asm in accelerator has it as min :') 

EDIT-2: future improvements is to check if it's lower than <60 then display sec or seconds or hours (food for thought)

EDIT-3: nvm about edit 2 because we're only doing displaying min:sec

Reply:
Indeed, it's like this in the mockup.